workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/OAuth2Authorize/OAuth2AuthorizeOutlet.tsx

```diff
@@ -16,6 +16,8 @@ import useAccounts from '../ProductPurchase/hooks/useAccounts';
 type Schema = z.infer<typeof zodSchema.installProductSchema>;
 
 const OAuth2AuthorizeOutlet = () => {
+	const [loading, setLoading] = useState(true);
```

Comment:
The loading should come from useGetResourceInfo which is making the request for projects